### PR TITLE
Upload SARIF files by detekt

### PR DESCRIPTION
**Pull request by CommanderTvis**

### CODE REVIEW

File: .github/workflows/diktat.yml [hunk 1]
```diff
@@ -18,4 +18,9 @@ jobs:
           distribution: liberica
       - uses: gradle/gradle-build-action@v2.1.5
         with:
-          arguments: diktatCheckAll --stacktrace
+          arguments: diktatCheckAll -Ddiktat.githubActions=true --stacktrace
+#      - uses: github/codeql-action/upload-sarif@v2
+#        if: ${{ always() }}
+#        with:
+#          sarif_file: /**/build/reports/diktat/diktat.sarif
+#          category: diktat
```
Comment (nbirillo) [inline .github/workflows/diktat.yml:26]: Why did you comment it?

**Review by nbirillo — APPROVED** [review]
Thank you!

File: .github/workflows/diktat.yml (hunk 1 above)
Comment (CommanderTvis) [reply .github/workflows/diktat.yml:26]: It doesn't work, Diktat seems to be unable to merge reports from multiple subprojects to one file.

Comment (nbirillo) [reply .github/workflows/diktat.yml:26]: Then we should delete it, isn't?

Comment (CommanderTvis) [reply .github/workflows/diktat.yml]: Done

**Review by nbirillo — APPROVED** [review]

**Pull request merged**